ruby/blueprints/installations/controller.rb

```diff
@@ -0,0 +1,25 @@
+module Blueprints::Installations
```

Comment:
please never declare a module as a concatenated namespace ... I can't remember the exact details but two options (the way you have it here compared to declaring the modules separately on successive lines) are not exactly equivalent and can cause naming problems ... maybe that's Rails only, but let's keep our style consistent.

Reply:
Good tip. Thanks. I'll change this one.